cogs/recruit.py

```diff
@@ -133,10 +138,11 @@ async def polling(self, ctx: commands.Context, action: str):
 
     @commands.hybrid_command(name="purge", with_app_command=True, description="Clear queue")
     @app_commands.guilds(configInstance.data.guild)
-    @commands.has_permissions(administrator=True)
     async def purge(self, ctx: commands.Context):
         await ctx.defer()
 
+        is_manager(ctx=ctx)
```

Comment:
`@commands.has_role(configInstance.data.manager_role_id)` is the right way to do this.

Reply:
In fact, we should update all of the 'isrecruiter' checks to use `@commands.has_role(configInstance.data.recruiter_role_id)`

Reply:
Done!